packages/tasit-action/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tasit-action",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
Did you end up having to manually increment these, or did lerna do it for you?

Reply:
Lerna did